dmoj/tests/test_globs.py

```diff
@@ -91,10 +94,5 @@ def test_problem_roots(self):
             for problem in ex_cases:
                 self.assertRaises(KeyError, judgeenv.get_problem_root, problem)
 
-    def test_supported_problems(self):
-        with self.mock_problem_roots:
-            supported_problems = judgeenv.get_supported_problems(warnings=False)
-            self.assertEqual(list(sorted(self.supported_problems)), list(sorted(supported_problems)))
```

Comment:
why don't we keep those test saparated?

Reply:
`problem_roots` is not populated until `get_supported_problems` is called

Reply:
ah, okay then